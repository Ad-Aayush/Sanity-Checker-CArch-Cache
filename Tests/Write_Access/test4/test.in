cache_sim enable ./Tests/Write_Access/test4/config.txt
load ./Tests/Write_Access/test4/input.s
run
cache_sim dump cache.dump
exit