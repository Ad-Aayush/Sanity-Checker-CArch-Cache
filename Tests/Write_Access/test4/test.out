D-cache statistics: Accesses=9, Hit=2, Miss=7, Hit Rate=0.22